cloudwatch-logs-to-logstash/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $TEST_FUNC_NAME ]; then
+    echo 'TEST_FUNC_NAME is unset';
+    exit 1
+fi
+
+if [ -z $PROD_FUNC_NAME ]; then
+    echo 'PROD_FUNC_NAME is unset';
+    exit 1
+fi
+
+cd lambda
+npm install
+
+npm test
+
+# TODO get or create config.json
```

Comment:
Opted for [manual deploy](https://github.com/guardian/grid/commit/ac5e89684d94a0e2aafb5d97a32d715b75fd0175) in this instance as this lambda won't update very often (if at all) as it just extracts json, then pushes to kinesis.